### PR TITLE
docs: update readme to v5.0

### PR DESCRIPTION
**Pull request by hund030**

### CODE REVIEW

File: hello-world-bot-with-tab/README.md [hunk 1]
```diff
@@ -1,14 +1,54 @@
...
+- From VS Code: 
+    1. Sign into Azure by clicking the `Sign in to Azure` under the `ACCOUNTS` section from sidebar.
+    1. Click `Provision in the Cloud` from `DEPLOYMENT` section or open the command palette and select: `Teams: Provision in the Cloud`.
+    1. Click `Deploy to the Cloud` or open the command palette and select: `Teams: Deploy to the Cloud`.
+
+- From TeamsFx CLI:
+    1. Run command: `teamsfx account login azure`.
+    1. Run command: `teamsfx provision --env dev`.
+    1. Run command: `teamsfx deploy --env dev`.
+
```
Comment (summzhan) [inline hello-world-bot-with-tab/README.md:40]: need to add ### Preview the app in Teams

File: hello-world-bot-with-tab/README.md [hunk 2]
```diff
@@ -1,14 +1,54 @@
...
+## Add Single Sign On feature
```
Comment (summzhan) [inline hello-world-bot-with-tab/README.md:41]: Can you change the title to "## Advanced usage of this sample"
Then followed with "### Add Single Sign On feaure"

File: hello-world-tab-with-backend/README.md [hunk 3]
```diff
@@ -1,77 +1,63 @@
...
+
 Microsoft Teams supports the ability to run web-based UI inside "custom tabs" that users can install either for just themselves (personal tabs) or within a team or group chat context.
 
 Hello World Tab with Backend shows you how to build a tab app with an Azure Function as backend, how to get user login information with SSO and how to call Azure Function from frontend tab.
 
 ![Hello World Tab](images/helloWorld-tab-with-backend.gif)
+
+## This sample illustrates
+
+- Use Teams Toolkit to create a Teams tab app.
```
Comment (summzhan) [inline hello-world-tab-with-backend/README.md:13]: "How to use xxx"

File: hello-world-bot-with-tab/README.md [hunk 4]
```diff
@@ -1,14 +1,54 @@
...
 
-Microsoft Teams apps bring key information, common tools, and trusted processes to where people increasingly gather, learn, and work.Apps are how you extend Teams to fit your needs. Create something brand new for Teams or integrate an existing app.
+This is a simple hello world application with both Bot and Tab capabilities.
 
-There are multiple ways to extend Teams, so every app is unique. Some only have one capability, while others have more than one feature to give users various options. For example, your app can display data in a central location, that is, the tab and present that same information through a conversational interface, that is, the bot.
+## This sample illustrates
 
-[What is Teams app capabilities](https://aka.ms/teamsfx-capabilities-overview)
+- Use Teams Toolkit to create a Teams app with both Bot and Tab capabilities.
```
Comment (summzhan) [inline hello-world-bot-with-tab/README.md:9]: "How to use xxx"

**Review by summzhan — CHANGES_REQUESTED** [review]
comment in line

File: hello-world-bot-with-tab/README.md [hunk 5]
```diff
@@ -1,14 +1,54 @@
...
+
+Microsoft Teams provides a mechanism by which an application can obtain the signed-in Teams user token to access Microsoft Graph (and other APIs). Teams Toolkit facilitates this interaction by abstracting some of the Azure Active Directory (AAD) flows and integrations behind some simple, high-level APIs. This enables you to add single sign-on (SSO) features easily to your Teams application.
+
+Please follow this [document](https://aka.ms/teamsfx-add-sso) to add single sign on for your project.
+
+## Version History
+
+|Date| Author| Comments|
+|---|---|---|
+|Dec 9, 2022| hund030 | update to support Teams Toolkit v5.0.0|
```
[the remark this answers is not in the document]
Comment (hund030) [reply hello-world-bot-with-tab/README.md:51]: This sample is for v5 only and there is no v4 history for it.

File: hello-world-bot-with-tab/README.md [hunk 6]
```diff
@@ -1,14 +1,54 @@
-# More Information
+# Getting Started with Hello World Bot with Tab
 
-You have a new Teams project scaffolded! To understand more about the structure of the project, you can read the readme files listed below to get further information.
+A bot, chatbot, or conversational bot is an app that responds to simple commands sent in chat and replies in meaningful ways. Examples of bots in everyday use include: bots that notify about build failures, bots that provide information about the weather or bus schedules, or provide travel information. A bot interaction can be a quick question and answer, or it can be a complex conversation. Being a cloud application, a bot can provide valuable and secure access to cloud services and corporate resources.
```
[the remark this answers is not in the document]
Comment (hund030) [reply hello-world-bot-with-tab/README.md]: Updated

File: hello-world-bot-with-tab/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (summzhan) [reply hello-world-bot-with-tab/README.md:61]: Then you should make it clear in Comments column. For example use words like "Onboard sample in Teams Toolkit V5.0.0"

[the remark this answers is not in the document]
Comment (hund030) [reply hello-world-bot-with-tab/README.md]: Good suggestion. Updated.

**Review by summzhan — APPROVED** [review]

**Pull request merged**